pass_import.py

```diff
@@ -1080,6 +1081,17 @@ def _compose_url(entry):
             url = "%s://%s" % (sub.get(protocol, protocol), url.strip())
         return url
 
+    @staticmethod
+    def _human_date(date):
+        """Return the date in human readable format"""
+        try:
+            if date[-1:] == '\x00':
```

Comment:
What is the purpose of this, the date has already been decoded and is not in hex any-more.

Reply:
I just double checked and hexdumped the result without the code in this PR, and the result includes an ending zero. The YAML parser does not remove it, and it’s working Ok because that zero is explicitly included in the original file exported from Apple’s Keychain app.
I would agree it looks like a patch. Do you know of a more proper way to handle it?

BTW, kudos for your work replacing my original code with the YAML parser; elegant work.